xave-finance/halodao-interface-archive
# Standard currency & balance formatting on pools page + added Swap & Vote menu to navbar

## Description of Changes

- HDF-142: Use balancer `formatNumber` function for formatting currencies and token balances throughout the app
- HDF-131: Create a "Swap" tab that when clicked redirects to the balancer swap page
- HDF-132: Create a "Vote" tab that when clicked redirects to our halodao snapshot page



## How to test

Check /pools page and verify that $ prices and HALO & BPT balances are showing up fine.

Swap and Vote tabs/menu added on navbar, and redirecting to correct url 

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDF-142, https://halodao.atlassian.net/browse/HDF-131, https://halodao.atlassian.net/browse/HDF-132)

### Developer Checklist:

- [x] My branch conforms with our naming convention i.e. `feature/BGM-XXX-description`

### Reviewers Checklist:

- [ ] Code style confirms to typescript rules
